src/test/java/com/climatetree/places/model/PlaceInfoTest.java

```diff
@@ -0,0 +1,85 @@
+package com.climatetree.places.model;
```

Comment:
If you incorporate the mapper class, You would then refactor this class to test that mapper class and not the models. currently Sonar has been configured to not test the models.

Reply:
I didn't realize models would be skipped. I added a Mapper class that handles conversions between objects